lib/iris/__init__.py

```diff
@@ -170,16 +170,21 @@ def __init__(self, cell_datetime_objects=False, netcdf_promote=False,
         encounters a GRIB message which uses a template not supported
         by the conversion.
 
+        The option `netcdf_no_unlimited` controls whether the netCDF saver
+        chooses the leading dimension of a cube to make unlimited or not in
+        the resulting netCDF file.
```

Comment:
This could do with mentioning that it's only the *default* behaviour that is affected.

How about: The option `netcdf_no_unlimited` controls whether the default behaviour of the netCDF saver is to make the leading dimension unlimited.